src/config.rs

```diff
@@ -16,6 +16,21 @@ pub struct Config {
     pub skip_setup: bool,
 }
 
+// Provide a default implementation for tests only
+#[cfg(test)]
+impl Default for Config {
+    fn default() -> Self {
```

Comment:
Implementing default only for tests is confusing.
Better create another factory, like Config::test()